rdkit/Chem/ChemUtils/AlignDepict.py

```diff
@@ -2,10 +2,8 @@
 #  Copyright (C) 2006 Greg Landrum
 #  This file is part of RDKit and covered by $RDBASE/license.txt
 #
-
-
-import argparse
 import sys
+import argparse
```

Comment:
Are you using an automated tool for sorting the imports?

Reply:
Yes, I used the automatic import sorting in both VSCode and PyCharm, but I fixed this one manually.